.travis.yml

```diff
@@ -0,0 +1,24 @@
+# @Date:   2017-03-20T16:36:18-04:00
+# @Project: scatt-jacc-caj
+# @Filename: travis.yml
+# @Last modified time: 2017-03-20T16:53:05-04:00
+
+#defining the language used
+language: java
+
+script:
+#installing dependencies
+- ./gradlew assemble
+#calling default script
+- ./gradlew check

```

Comment:
You may be correct as the defaults may automatically be what I am doing here, I was unsure so I did this as a means of being 'safe' rather than sorry. But if you think gradlew build would perform the same functionality I'd be happy to change it up.
Here's where I came to my conclusions (same as below):
https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle